network/alsp/manager/manager.go

```diff
@@ -308,10 +317,36 @@ func (m *MisbehaviorReportManager) onHeartbeat() error {
 				return record, fmt.Errorf("illegal state: spam record %x has non-positive decay %f", id, record.Decay)
 			}
 
+			// TODO: this can be done in batch but at this stage let's send individual notifications.
```

Comment:
There should be more documentation here explaining how falling below `model.DisallowListingThreshold` would make the node temporarily delisted from the network and would cut off communication with other nodes at the libp2p layer.

Reply:
Added a documentation with a diagram here: https://github.com/onflow/flow-go/pull/4441/commits/473153ba4144829cb90f98b45cda26fe426c0f2e